schema/ListedLicense.xsd

```diff
@@ -7,7 +7,7 @@
 				This format is internal to the SPDX legal team and is subject to change.  For stable output formats, see <xhtml:a href="https://github.com/spdx/license-list-data">license-list-data</xhtml:a>.
 			</xhtml:p>
 			<xhtml:p>
-				Licensed under CC-BY-4.0.
+				SPDX-License-Identifier: CC0-1.0.
```

Comment:
The trailing period here is not part of a valid license expression.  Can we drop it?

Reply:
fixed, good catch!